### DISCUSSION TITLE
How to install Vue Devtool for Tauri App

### DISCUSSION THREAD

**Discussion by vanminhquangtri** [tauri-apps/tauri#7465] [Q&A] [answered]
Hi Brothers, I see this post is from 2021 so I hope something change today:
https://github.com/tauri-apps/tauri/discussions/2685

Yes, my question is Is there away to install Vue Devtool for Tauri App on Window and Mac OS.
I am using Vue 3 + @tauri-apps/api version 1.4.0.
The extension I want to install is https://chrome.google.com/webstore/detail/vuejs-devtools/nhdogjmejiglipccpnnnanhbledajbpd.
This extension help much on debug app rathan console.log.

Thank you.

**Comment by FabianLars** [comment, marked as answer]
The issue you linked is still accurate. No extensions in tauri apps. And _when_ it's supported it will be Windows-only too.

You still can use vue devtools by using their standalone version: <https://devtools.vuejs.org/guide/installation.html#standalone>.
Depending on the rest of your frontend setup there may be integrations for that, like the Nuxt devtools or https://github.com/webfansplz/vite-plugin-vue-devtools for the vitejs bundler.
